README.md

```diff
@@ -15,15 +15,19 @@ Currently uses HTTP as the transport layer, with plans to add other options as t
 
 ## Usage example
 
-Initialize similar to a regular microservice, but pass a `JsonRpcService` as the strategy option:
+Initialize similar to a regular microservice used in a hybrid application, but pass a `JsonRpcService` as the strategy option:
```

Comment:
Can we keep both examples i.e. standalone and hybrid-app?

Something like: " You can also use it in a hybrid application on the same port as a HTTP server by passing the http adapter, like so:"